front/app/modules/commercial/content_builder/admin/components/CraftComponents/Container/index.tsx

```diff
@@ -16,7 +18,10 @@ const Container: UserComponent = ({ children }) => {
 
 Container.craft = {
   rules: {
-    canMoveIn: (nodes) => nodes.every((node) => node.data.type !== Container),
+    canMoveIn: (nodes) =>
+      nodes.every(
+        (node) => node.data.type === Text || node.data.type === Image
```

Comment:
I previously broke this rule by allowing two and three columns to be nested in a container. Fixed now!